packages/fcalclusterer/package.py

```diff
@@ -27,6 +27,9 @@ class Fcalclusterer(CMakePackage):
     depends_on('root +unuran +math')
     depends_on('dd4hep')
 
+    # CMAKE_INSTALL_PREFIX is overwritten by the package
+    patch("install.patch")
```

Comment:
@vvolkl I need some advice on how to limit when to apply the patch, when I fix this in FCalClusterer directly.

Reply:
@andresailer if it is fixed after `1.0.1`, then that should be

`patch("install.patch", when="@:1.0.1")`

Reply:
Thanks!